### PR TITLE
Update arcade

### LINKED ISSUE #577: Update .NET SDK version
This requires fixing an Arcade bug which breaks publishing RIT. /cc @jmarolf

https://github.com/dotnet/arcade/pull/2003/files#diff-ca11509da9be27e6202dd06024395ed4

After this we should be able to start regularly consuming Arcade updates and that way remain up to date.

### PR DESCRIPTION
**Pull request by RikkiGibson**
Closes #577 
Continuation of #574 
Related to dotnet/arcade#4341

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,8 +1,13 @@
 {
   "tools": {
-    "dotnet": "2.1.401"
+    "dotnet": "3.0.100",
+    "runtimes": {
+      "dotnet": [
+        "2.1.11"
```
Comment (JoeRobich) [inline global.json]: Is this for the xunit test runner?

Comment (RikkiGibson) [reply global.json]: Actually good question. @jmarolf could we maybe just delete this and automatically use the runtime included with 3.0.100 SDK?

Comment (jmarolf) [reply global.json]: yep

Comment (RikkiGibson) [reply global.json]: Will probably do this in a future PR

**Pull request merged**